### DISCUSSION TITLE
Where are the Substrait visualizers?

### DISCUSSION THREAD

**Discussion by nchammas** [substrait-io/substrait#608] [Q&A] [answered]
The [homepage][1] has this interesting line:

[1]: https://substrait.io

> The text version of the Substrait plan allows you to quickly see how a plan functions without needing a visualizer (although there are Substrait visualizers as well!).

Where are these visualizers? A cursory search of the site and repo only yielded [this issue][2], but the visualization tool mentioned there is not from this project. Is that what this line is referencing?

[2]: https://github.com/substrait-io/substrait/issues/444

**Comment by EpsilonPrime** [comment, marked as answer]
The text format is in the substrait-cpp repo.  Substrait-fiddle.com can show it to you in the web
